source-jira/tests/snapshots/spec.stdout.json

```diff
@@ -2,124 +2,104 @@
   "protocol": 3032023,
   "configSchema": {
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Jira Spec",
+    "type": "object",
+    "required": [
+      "api_token",
+      "domain",
+      "email"
+    ],
     "additionalProperties": true,
     "properties": {
       "api_token": {
-        "airbyte_secret": true,
-        "description": "Jira API Token. See the docs for more information on how to generate this key. API Token is used for Authorization to your account: https://go.estuary.dev/Bk7fII",
-        "order": 0,
+        "type": "string",
         "title": "API Token",
-        "type": "string"
+        "description": "Jira API Token. See the docs for more information on how to generate this key. API Token is used for Authorization to your account: https://go.estuary.dev/Bk7fII",
+        "airbyte_secret": true,
+        "order": 0
       },
       "domain": {
-        "description": "The Domain for your Jira account, e.g. estuary.atlassian.net, estuary.jira.com, jira.your-domain.com",
+        "type": "string",
+        "title": "Domain",
         "examples": [
           "<your-domain>.atlassian.net",
           "<your-domain>.jira.com",
           "jira.<your-domain>.com"
         ],
-        "order": 1,
-        "title": "Domain",
-        "type": "string"
+        "description": "The Domain for your Jira account, e.g. estuary.atlassian.net, estuary.jira.com, jira.your-domain.com",
+        "order": 1
       },
       "email": {
-        "description": "The user email for your Jira account which you used to generate the API token. This field is used for Authorization to your account by BasicAuth.",
-        "order": 2,
+        "type": "string",
         "title": "Email",
-        "type": "string"
-      },
-      "enable_experimental_streams": {
```

Comment:
It seems like there are some fields that are being removed in the new spec, but it's hard to tell from this diff. Removal of fields from the spec is not backward compatible and is something we'll want to think carefully about.

Can you comment more on what is changing in the spec snapshot? A link to a [jsondiff](https://github.com/zgrossbart/jdd?tab=readme-ov-file#load-my-json-data-from-the-internet) that is easier to visually parse would probably suffice here.